wsmaster/che-core-sql-schema/src/main/resources/che-schema/7.0.0-beta3.0/1__add_devfile.sql

```diff
@@ -0,0 +1,260 @@
+--
+-- Copyright (c) 2012-2019 Red Hat, Inc.
+-- This program and the accompanying materials are made
+-- available under the terms of the Eclipse Public License 2.0
+-- which is available at https://www.eclipse.org/legal/epl-2.0/
+--
+-- SPDX-License-Identifier: EPL-2.0
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+
+
+
+-- add devfile table
+CREATE TABLE devfile (
```

Comment:
We used to use `che_` prefix for Che tables.

Reply:
You're right that we do not support Codenvy anymore. But we started without `che_` prefix. And then we faced an issue during moving tables from codenvy to che, and we solved it with product-specific prefixes. I think it makes sense to keep using prefixes since it may help us in the future and it costs nothing for use.

Reply:
@skabashnyuk 

> about prefixing tables with che_. I would say I prefer not to do that if we have no strong need to do that. Because it makes the table name longer, we have no strong need to do that, ~ 50% of tables do not have that prefix, less character in the name means less time and memory to process query. @sleshchenko @metlos wdyt?

If table name essentially influences on productivity then we should care about it and use shorter names as possible, for example `devfile_cmd_attrs` instead of `devfile_command_attributes`
I guess it is not essentially comparing to that fact that we remove all objects with generated ids instead of applying changes on them each time when the root object is updated. For example, we remove all of WorkspaceConfig, WorkspaceConfigAttributes, ServerConfig, ServerConfigImpl, VolumeConfig, ... each time when one attribute in Workspace is changed and no actual changes in nested objects.

I'm OK with using table name without prefix since it's an upstream project. Since downstream projects may prefix their tables to avoid collisions and further issues with moving a table to upstream (it's what we did with Codenvy).